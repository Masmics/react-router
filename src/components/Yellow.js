import React from 'react';

export default function Yellow() {
  return (
    <>
      <h1>Yellow!</h1>
      <div style={{ width: '100px', height: '100px', backgroundColor: '#ffff00' }} />
    </>
  );
}
