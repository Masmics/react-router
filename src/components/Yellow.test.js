import React from 'react';
import { shallow } from 'enzyme';
import Yellow from './Yellow';

describe('Yellow component', () => {
  it('renders the yellow', () => {
    const wrapper = shallow(<Yellow />);
    expect(wrapper).toMatchSnapshot();
  });
});
