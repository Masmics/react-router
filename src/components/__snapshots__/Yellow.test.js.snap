// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Yellow component renders the yellow 1`] = `
<Fragment>
  <h1>
    Yellow!
  </h1>
  <div
    style={
      Object {
        "backgroundColor": "#ffff00",
        "height": "100px",
        "width": "100px",
      }
    }
  />
</Fragment>
`;
